### PR TITLE
Ghex rma

### PR DESCRIPTION
**Pull request by boeschf**
Add GHEX rma capabilities to gtbench

### CODE REVIEW

**Comment by boeschf** [comment]
CI is somehow unable build with ghex. Could it be that the container needs updating?

**Comment by fthaler** [comment]
Please retry after merging with the master branch, there might have been some undiscovered incompatibilities with dependency versions.

**Comment by boeschf** [comment]
still seems to use an old version of GHEX

**Comment by fthaler** [comment]
You have to set the preferred GHEX version here: https://github.com/GridTools/gtbench/blob/f1b10e9399a763bf4f0e3ee500e5e64891b2e9e4/runtime/ghex_comm/CMakeLists.txt#L8

**Comment by boeschf** [comment]
thanks for the hint! I set the correct hash now, but the hip tests fail - I think this is independent of the ghex stuff.

File: README.md [hunk 1]
```diff
@@ -5,82 +5,29 @@ The benchmark implements an advection-diffusion solver using the finite differen
...
-### GHEX
-
-For the GHEX runtime, installing the GHEX library is a further requirement. Note that this step can be skipped for single-node (single-process) runs or when using a different runtime than `ghex_comm` for inter-node communication (see further below for details about available runtimes). GHEX can be obtained using:
-
-```console
-$ git clone https://github.com/GridTools/GHEX.git
-```
-
-#### Dependencies
+Further external dependencies are listed below:
```
Comment (fthaler) [inline README.md:11]: Awesome, many thanks for updating! :1st_place_medal:

**Review by fthaler — APPROVED** [review]

**Comment by fthaler** [comment]
Resolves #63

**Pull request merged**